__tests__/app.test.js

```diff
@@ -337,3 +337,29 @@ describe("GET /api/users", () => {
       });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  it("should return 204 with no content", () => {
```

Comment:
It would be great to have a test here to check the database to ensure that comment has been removed. You already have a GET endpoint for comments - so try making a request to that to check that you get feedback that the comment is no longer accessible.